tests/integration/test_cli.py

```diff
@@ -44,6 +45,20 @@
 PRIVATE_KEY_FILE = KEY_DIR / "key.sec"
 
 
+def mock_input_wps_string(_: str):
+    """
+    Helper util to mock user input
+    """
+
+    id = "1"
+    token = "abcde"
+
+    pubkey = crypt4gh.keys.get_public_key(PUBLIC_KEY_FILE)
+
+    wps_string = id + ":" + encrypt(token, pubkey)
```

Comment:
```suggestion
    wps_string = id + "," + encrypt(token, pubkey)
```

Reply:
Same thing, the documentation says colon.

Reply:
https://github.com/ghga-de/epic-docs/blob/main/26-dracula-ant/technical_specification.md#work-order-token